src/Workspaces/Core/Portable/Formatting/FormattingOptions.cs

```diff
@@ -69,6 +69,21 @@ private static Option<T> CreateOption<T>(OptionGroup group, string name, T defau
             defaultValue: false,
             storageLocations: EditorConfigStorageLocation.ForBoolOption("insert_final_newline"));
 
+        /// <summary>
+        /// Default value of 120 was picked based on the amount of code in a github.com diff at 1080p.
+        /// That resolution is the most common value as per the last DevDiv survey as well as the latest
+        /// Steam hardware survey.  This also seems to a reasonable length default in that shorter
+        /// lengths can often feel too cramped for .Net languages, which are often starting with a
+        /// default indentation of at least 16 (for namespace, class, member, plus the final construct
+        /// indentation).
+        /// </summary>
+        internal static Option<int> PreferredWrappingColumn { get; } = new Option<int>(
+            nameof(FormattingOptions),
+            FormattingOptionGroups.NewLine,
+            nameof(PreferredWrappingColumn),
+            defaultValue: 120,
+            storageLocations: EditorConfigStorageLocation.ForInt32Option("dotnet_preferred_wrapping_column"));
```

Comment:
❗️ Need to remove the storage location for now. The feature/option can stay (so it's easy to hook up later to the final design), and the default can stay at 120.

Reply:
Is there a problem having the editorconfig option as well?  or is more that you want to design it out separately?

Reply:
It overlaps with an ongoing design that affects other teams. Rather than block the refactoring feature or end up with a duplicate setting, we decided to simply omit the storage/configurability and address that as a follow-up.